ide/css.editor/src/org/netbeans/modules/css/editor/csl/CssCompletion.java

```diff
@@ -1682,6 +1689,40 @@ public boolean visit(Node node) {
         } //switch
     }
 
+    @SuppressWarnings("AssignmentToMethodParameter")
+    public final int levenshteinDistance(
+            @NonNull String str1,
+            @NonNull String str2,
+            final boolean caseSensitive) {
```

Comment:
putting blank line after { would make it more readable

Reply:
This is a verbatim copy, so I'll keep it this way.